sdks/go/pkg/beam/core/graph/coder/stringutf8_test.go

```diff
@@ -120,3 +120,26 @@ func TestEncodeDecodeStringUTF8LP(t *testing.T) {
 		})
 	}
 }
+
+func FuzzEncodeDecodeStringUTF8LP(f *testing.F) {
```

Comment:
Is there a reason to keep the above test that is doing the same thing as this one? Same question for the tests in other files

Reply:
We could replace all of the encode/decode tests with fuzz tests that use the testValues as a seed corpus if we wanted to.

Reply:
I'd vote we do this rather than duplicating it - it will simplify any future test changes

Reply:
Hm, ok - I'd probably prefer that we split those out into their own test case so we're not double testing the same things, but I don't feel strongly about it - I'll approve and leave that up to you (and/or future reviewers)

Reply:
There's an argument for moving the fuzz tests for all of the coders to their own file for clarity

Reply:
What is the reasoning behind ignoring encoding failures in the fuzz tests instead of just failing them? If that small change was made there's no reason we couldn't replace the existing tests, right?

Reply:
I'm actually curious on this one as well - I'd initially assumed there were inputs that should fail when encoding, but I can't think of any. Are there inputs like that? If so, are there few enough that we can special case them?